### PR TITLE
31959 Update Vaccine Choose Facility Copy Doc Alert Message

### PR DESCRIPTION
**Pull request by vbahinwillit**
## Description
This PR updates content based on copy doc

## Original issue(s)
department-of-veterans-affairs/va.gov-team#31959

## Testing done
Unit testing

## Screenshots
localhost_3001_health-care_schedule-view-va-appointments_appointments_new-covid-19-vaccine-appointment_choose-facility (1)

## Acceptance criteria
- [ ] All test should pass

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/covid-19-vaccine/components/PlanAheadPage.jsx [hunk 1]
```diff
@@ -36,6 +36,10 @@ export default function PlanAheadPage() {
           If you’re eligible for a booster shot or additional dose, contact your
           VA health facility.
         </li>
+        <li>
```
Comment (jenniemc) [inline src/applications/vaos/covid-19-vaccine/components/PlanAheadPage.jsx:39]: This looks to be a duplicate bullet item from above

**Review by jenniemc — CHANGES_REQUESTED** [review]
Just need to remove the extra bullet item from `PlanAheadPage`. Everything else look good.

File: src/applications/vaos/covid-19-vaccine/components/PlanAheadPage.jsx (hunk 1 above)
Comment (vbahinwillit) [reply src/applications/vaos/covid-19-vaccine/components/PlanAheadPage.jsx:39]: Thanks. I don't know how that got there!

**Comment by jenniemc** [comment]
@vbahinwillit, here's a crazy thought. The original GH ticket has a yellow outline around the blue alert message "Some COVID-19 vaccines require 2 doses..." I'm wondering if it means to remove the message because it doesn't appear in the [copy doc](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/content/copy-docs/vaccine-choose-facility.md).

**Review by jenniemc — APPROVED** [review]
LGTM!

**Review by jenniemc — CHANGES_REQUESTED** [review]
Darn, looks like it is failing the `covid19-vaccine.cypress.spec.js`. Just got to remove the line that expects the text "Some COVID-19 vaccines require 2 doses"

**Review by ptouchton — CHANGES_REQUESTED** [review]
disregard my requested change...I just saw Jennie's review and it was the same thing

**Review by ptouchton — APPROVED** [review]

**Review by ptouchton — APPROVED** [review]

**Review by jenniemc — APPROVED** [review]
LGTM!

**Pull request merged**